homeassistant/components/homekit/const.py

```diff
@@ -43,7 +43,13 @@
 SERV_TEMPERATURE_SENSOR = 'TemperatureSensor'
 SERV_THERMOSTAT = 'Thermostat'
 SERV_WINDOW_COVERING = 'WindowCovering'
-
+SERV_CARBON_MONOXIDE_SENSOR = 'CarbonMonoxideSensor'
+SERV_CARBON_DIOXIDE_SENSOR = 'CarbonDioxideSensor'
+SERV_OCCUPANCY_SENSOR = 'OccupancySensor'
+SERV_CONTACT_SENSOR = 'ContactSensor'
+SERV_MOTION_SENSOR = 'MotionSensor'
+SERV_LEAK_SENSOR = 'LeakSensor'
+SERV_SMOKE_SENSOR = 'SmokeSensor'
```

Comment:
Please sort them alphabetically. The chars and device_classes as well.

Reply:
Follow it.
Furthermore, I also sort CHAR_CURRENT_HUMIDITY  and CHAR_CURRENT_POSITION, and this cause a conflict. Should I revert it, or would you resolve it?

Reply:
Try rebaseing it against `dev`: https://www.home-assistant.io/developers/development_catching_up